static/js/user_group_create.js

```diff
@@ -101,6 +115,7 @@ function create_user_group() {
     const data = {};
     const group_name = $("#create_user_group_name").val().trim();
     const description = $("#create_user_group_description").val().trim();
+    set_name(group_name);
```

Comment:
I think we need to call `reset_name` in the error handler too, if this is going to work.

Reply:
Ohh sorry, I missed that. Have added it now.